src/test/ui/borrowck/issue-63026.rs

```diff
@@ -0,0 +1,18 @@
+use std::collections::HashMap;
```

Comment:
Maybe add a comment explaining the previous issue?

Reply:
and a give this file a more descriptive name